scripts/004_migrate_add_theme.rb

```
@@ -0,0 +1,86 @@
+  insert_index = lines.index { |line| line.match?(/^footage_summary:/) }
+  insert_index ||= lines.index { |line| line.match?(/^videos:/) }
+
+  unless insert_index
+    puts "  ✗ No `footage_summary:` or `videos:` anchor found; skipping"
+    return false
+  end
+
+  lines.insert(insert_index, block, "\n")
+  new_content = lines.join
+
+  parsed = YAML.load(new_content, permitted_classes: [Date, Time, Symbol])
```

RuboCop (1.86.1)

[convention] Prefer using `YAML.safe_load` over `YAML.load`.

(Security/YAMLLoad)